packages/libs/error-stack/Cargo.toml

```diff
@@ -25,6 +34,8 @@ futures = ["dep:pin-project"]
 
 [package.metadata.docs.rs]
 all-features = true
+cargo-args = ["-Z", "unstable-options", "-Z", "rustdoc-scrape-examples=all"]
+targets = ["x86_64-unknown-linux-gnu"]
```

Comment:
We don't have any target-specific configuration, so we only need to build the default target. This helps docs.rs to avoid building 6 different targets. see [the blog post on this](https://blog.rust-lang.org/2020/03/15/docs-rs-opt-into-fewer-targets.html) and [their homepage](https://docs.rs/about/metadata).